### ISSUE TITLE
Subcategory: Need to create new subcategory pages

### ISSUE THREAD

**Issue by spaulenoff** [akcdotorg/akc.org#99]
- URL structure should be /content/[Category]/[sub-category]
- Will not change URL structure of individual pieces of content

**Comment by kylejbertelsen** [comment]
**Pages w/in a Category (status):**
/content/category-a/ (good)
/content/category-a/articles/  (good)
/content/category-a/videos/ (good)
/content/category-a/photo-galleries/ (good)
/content/category-a/subcategory-a-1/ (please add)
/content/category-a/subcategory-a-2/ (please add)

Single content views of subcategory-a-1(etc) pages will be (note the use of category-a, NOT subcategory-a-1):
/content/category-a/articles/article-title/
/content/category-a/videos/video-title/
/content/category-a/photo-galleries/gallery-title/

**Comment by toshan2014** [comment]
@spaulenoff - please confirm whether the slider at the top will contain the same items as the listed items ( in the boxes ) in the main area

**Comment by spaulenoff** [comment]
@toshan2014 Confirmed the same content might appear in both places. The Slider at the top will feature the featured/newest content i believe.

**Comment by spaulenoff** [comment]
@toshan2014 Apologies but we will not need the slider up top. The hero will be static like one of these pages: http://develop2.akc.org/content/westminster-dog-show/articles/ just with the content not sorted by content type

![image](https://cloud.githubusercontent.com/assets/13648969/13585327/3236f01a-e489-11e5-9726-1a059ef08ff6.png)

**Comment by spaulenoff** [comment]
@toshan2014 Close, but we want the content to not be broken up by type like it is on http://develop2.akc.org/content/westminster-dog-show/live-coverage/
![image](https://cloud.githubusercontent.com/assets/13648969/13638616/180db432-e5db-11e5-8af9-ba9ca9d8bb41.png)

We want all content to be mixed in together like in the below screenshot, with the newest content moving to the top
![image](https://cloud.githubusercontent.com/assets/13648969/13638576/e4b069b8-e5da-11e5-8569-f6d2289e900e.png)

Can we have also so the the H1 Tag only has the subcategory Title? Like So:
![image](https://cloud.githubusercontent.com/assets/13648969/13638658/4968e128-e5db-11e5-9fa8-9fdd3780ee5d.png)

**Comment by toshan2014** [comment]
@spaulenoff changes made - please confirm

**Comment by spaulenoff** [comment]
@toshan2014 Need to rework this so it doesn't affect category pages such as http://develop2.akc.org/content/westminster-dog-show/

**Comment by toshan2014** [comment]
Please check again : 
https://develop2.akc.org/content/westminster-dog-show/ ( unaffected )
https://develop2.akc.org/content/westminster-dog-show/live-coverage/ ( new subcat page )

**Comment by spaulenoff** [comment]
@elieo @stevepessah The new subcategory page can be found here:
http://develop2.akc.org/content/westminster-dog-show/live-coverage/
Please QA and please let us know if there is anything that you feel needs to still be fixed
http://develop2.akc.org/content/westminster-dog-show/live-coverage/

I checked against the mockup, and besides the featured content header (which Elie said is ok to leave out), it seems good to me

**Comment by elieo** [comment]
@spaulenoff 
- "media" in the header needs to be removed (also on category pages) 
- The left navigation is not collapsing in a "tree style navigation" 
- Mobile - The hero image looks cut off on the sides (we may need to have a separate hero image for mobile) 
- Mobile - Where is the menu? 
- Mobile - Looks like the social buttons are showing on the category pages for iphone 4 but not iphone 6. (let's remove social buttons on category pages) 
- The tablet version of this page could use lots of improvements.

**Comment by spaulenoff** [comment]
@elieo Answer to points above:
- "media" in the header needs to be removed (also on category pages) @toshan2014  please remove
- The left navigation is not collapsing in a "tree style navigation" (Lets discuss with @kylejbertelsen per https://github.com/akcdotorg/akc.org/issues/98)
- Mobile - The hero image looks cut off on the sides (we may need to have a separate hero image for mobile) Let's discuss
- Mobile - Where is the menu? (This is a separate Ticket: https://github.com/akcdotorg/akc.org/issues/52)
- Mobile - Looks like the social buttons are showing on the category pages for iphone 4 but not iphone 6. (let's remove social buttons on category pages) @skt2 Lets remove mobile social buttons on category pages
- The tablet version of this page could use lots of improvements.  (Do we have mockups for Tablet?)

**Comment by toshan2014** [comment]
@spaulenoff - Removed 'Media' from title

**Comment by spaulenoff** [comment]
@toshan2014 Thank you, can you please add a field to the cms to upload mobile hero images? The Recommended size should be 375px X 280px
@skt2 Same thing for category pages on mobile ^

**Comment by toshan2014** [comment]
This uses the same field as /category (in Category Page channel). Once this is done for category, I'll just modify the field name for /subcategory. Please reassign to me once done in /category.

**Comment by spaulenoff** [comment]
@elieo @stevepessah Updates below
- "media" in the header needs to be removed (also on category pages) **Updated**
- The left navigation is not collapsing in a "tree style navigation" **Updated**
- Mobile - The hero image looks cut off on the sides (we may need to have a separate hero image for mobile) **@skt2 will work on this**
- Mobile - Where is the menu? **Part of #52 being worked on by Kyle**
- Mobile - Looks like the social buttons are showing on the category pages for iphone 4 but not iphone 6. (let's remove social buttons on category pages) **Updated**
- The tablet version of this page could use lots of improvements.  **We need to discuss**

**Comment by spaulenoff** [comment]
@elieo @stevepessah Please QA on Dev2 http://develop2.akc.org/content/westminster-dog-show/live-coverage/

Please note the following issues are being worked on still: 
- Mobile - The hero image looks cut off on the sides (we may need to have a separate hero image for mobile)
- Mobile
  - Where is the menu? **Part of #52 being worked on by Kyle**

**Comment by spaulenoff** [comment]
@toshan2014 One More update: Can we change the Sub Category pages to match the following mockup?
![pasted image at 2016_03_16 10_33 am](https://cloud.githubusercontent.com/assets/13648969/13817081/3997b7e2-eb67-11e5-9856-3d32e2c12c44.png)
http://develop2.akc.org/content/dog-training/unwanted-behaviors/

So the Category Name will show in the H1 and the Sub category name will be in the H2

**Comment by kylejbertelsen** [comment]
For this latest change, I added an hour to the estimate.

**Comment by toshan2014** [comment]
@spaulenoff please qa : https://develop2.akc.org/content/dog-training/unwanted-behaviors/

**Comment by toshan2014** [comment]
please qa on stage : http://stage-ng.akc.org/content/westminster-dog-show/live-coverage/

**Comment by toshan2014** [comment]
pf

http://develop2.akc.org/content/westminster-dog-show/live-coverage/

**Comment by spaulenoff** [comment]
@toshan2014 Verified on Prod. Please add ticket with time tracking for this ticket

**Comment by toshan2014** [comment]
Time spent: around 20h
